developments_build/sql/_spatial.sql

```diff
@@ -156,4 +156,4 @@ SELECT
 INTO SPATIAL_devdb
 FROM DRAFT_spatial a
 LEFT JOIN lookup_geo b
-ON a.geo_boro||a.geo_censustract2010 = b.borocode||b.ct2010;
\ No newline at end of file
+ON a.geo_boro||a.geo_censusblock2010||a.geo_censustract2010 = b.bctcb2010
```

Comment:
I think this should be `a.geo_boro||a.geo_censustract2010||a.geo_censusblock2010`